core/src/playerglobal/README.md

```diff
@@ -0,0 +1,40 @@
+# Ruffle playerglobal
```

Comment:
I think this directory should be located under `core/src/avm2`, since it's only relevant for AVM2.

Reply:
How about placing this in a new toplevel `compiler/src/avm2/` directory? This way `tests` could benefit from this as well.

Reply:
I see this as being an implementation detail of `core` - do you have a scenario in mind where `tests` would need to access it?

Reply:
Sure, I plan migrating the tests (both AVM1 and AVM2) to pure ActionScript, rather than the binary `.fla` files. It'll make writing tests much more convenient.